### PR TITLE
Feature: Start to run `digestabot` nightly to bump digests along a tag.

### PR DESCRIPTION
**Pull request by mattmoor**
:gift: This creates a digestabot workflow that will send a nightly PR to bump digests to the latest for a given tag.

This also changes a bunch of references to use explicit digests in a form we will bump.

/kind feature

### CODE REVIEW

**Comment by mattmoor** [comment]
I'm going to pin a bunch more places, so ignore this for the time being.

File: Makefile [hunk 1]
```diff
@@ -82,4 +82,4 @@ $(foreach pkg,$(PKGLIST),$(eval $(call build-package,$(pkg))))
 .build-packages: ${PACKAGES}
 
 dev-container:
-	docker run --privileged --rm -it -v "${PWD}:${PWD}" -w "${PWD}" cgr.dev/chainguard/sdk:latest
+	docker run --privileged --rm -it -v "${PWD}:${PWD}" -w "${PWD}" ghcr.io/wolfi-dev/sdk:latest@sha256:515a2c5072753f81ce2cbb81bda54b035aefcdb41d7249070009fc018fecd4c9
```
Comment (mattmoor) [inline Makefile:85]: @kaniini This is the reference I was referring to, which I pointed to `wolfi-dev/sdk` instead.  Please LMK if this was in error!

File: .github/workflows/dag-push-production.yaml [hunk 2]
```diff
@@ -134,6 +134,7 @@ jobs:
             --cpu=30 --ram=100Gi \
             --bucket=${BUCKET} \
             --src-bucket=${SRC_BUCKET} \
+            --sdk-image ghcr.io/wolfi-dev/sdk:latest@sha256:515a2c5072753f81ce2cbb81bda54b035aefcdb41d7249070009fc018fecd4c9 \
```
Comment (mattmoor) [inline .github/workflows/dag-push-production.yaml:137]: Thanks to @imjasonh for this pointer 🙏

File: Makefile (hunk 1 above)
Comment (imjasonh) [reply Makefile:85]:
```
crane config cgr.dev/chainguard/sdk | jq .created
"2023-01-19T06:08:12Z"
```

Kill it with fire 🔥

**Review by imjasonh — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (kaniini) [reply Makefile:85]: Nah this is fine

**Review by kaniini — APPROVED** [review]

**Comment by mattmoor** [comment]
Did a trial run here: https://github.com/wolfi-dev/os/actions/runs/4854190506/jobs/8651250356

**Pull request merged**